contracts/registry/src/registry.rs

```diff
@@ -335,11 +335,12 @@ impl SBTRegistry for Contract {
         SbtTokensEvent { issuer, tokens }.emit_renew();
     }
 
-    /// Revokes SBT by burning it.
+    /// Revokes SBT.
     /// Must be called by an SBT contract.
-    /// Must emit one of `Revoke` or `Burn` event.
+    /// Must emit `Revoke` event.
+    /// Must emit also emit `Burn` event if the SBT tokens are burned (removed).
     /// Returns true if a token is a valid, active SBT. Otherwise returns false.
```

Comment:
How about the return value? We are passing multiple tokens to the method

Reply:
true, let's remove return value